qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -608,7 +608,7 @@
     <item>
      <widget class="QComboBox" name="comboBox_5">
       <property name="sizePolicy">
-       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
```

Comment:
What's the reasoning behind this? The commit just says the fixed size is 'far too small', but for an integer value that will almost certainly be <= 16, a box that can fit three digits should be fine. Letting it grow to larger sizes without good reason just creates empty space in the UI.